recipe/meta.yaml

```
@@ -1,5 +1,5 @@
 {% set name = "make-it-sync" %}
-{% set version = "2.0.0" %}
+{% set version = "2.1.1" %}
 {% set python_min = "3.7" %}
```

```suggestion
{% set python_min = "3.9" %}
```

Need to bump to 3.9 given https://github.com/gordonwatts/make-it-sync/compare/2.0.0...2.1.1